lib/presets.js

```diff
@@ -27,9 +27,42 @@ const path = require('path');
 const updateJSON = require('update-json');
 const fs = require('fs');
 const semver = require('semver');
-const replace = require('replace-in-file');
+const replaceInFile = require('replace-in-file');
 const markdown = require('./markdown');
 
+/**
+ * @summary Replaces the contents of a file using a regular expression
+ * @function
+ * @private
+ *
+ * @param {String} file - target filename
+ * @param {RegExp} pattern - pattern used in str.replace
+ * @param {String} replacement - replacement used in str.replace
+ * @param {Function} callback - callback (error)
+ */
+const replace = (file, pattern, replacement, callback) => {
+  async.waterfall([
+    (done) => {
+      fs.readFile(file, 'utf8', done);
+    },
+
+    (contents, done) => {
+      if (pattern.test(contents)) {
+        done(null, contents);
+      } else {
+        done(new Error(`Regex does not match ${file}`));
```

Comment:
To match the function parameter names, should this say "Pattern" instead of "Regex"?

Reply:
Will modify that.